src/core_modules/capture-core/components/WidgetEventSchedule/hooks/useDetermineSuggestedScheduleDate.js

```diff
@@ -51,17 +51,36 @@ type Props = {
     enrolledAt: string,
     occurredAt: string,
     initialScheduleDate: string,
-    eventData: Array<Object>
+    eventData: Array<Object>,
+    hideDueDate?: boolean
 }
+
+const calculateSuggestedDateFromStart = ({
+    generatedByEnrollmentDate,
+    displayIncidentDate,
+    enrolledAt,
+    occurredAt,
+    minDaysFromStart,
+}: Object) => {
+    let suggestedScheduleDate;
```

Comment:
NIT: Use `const` instead of `let`.
```
    const suggestedStartDate = (generatedByEnrollmentDate || !displayIncidentDate)
        ? enrolledAt : occurredAt;
    const suggestedScheduleDate = moment(suggestedStartDate).add(minDaysFromStart, 'days').format();
    return suggestedScheduleDate;
```

Reply:
thanks @simonadomnisoru but I think I'd keep the `let` here to emphasise on the logic of calculating the suggested date.